multirun/def.bzl

```diff
@@ -60,9 +60,11 @@ def _multirun_impl(ctx):
             tag = tag,
             path = exe.short_path,
         ))
+    if ctx.attr.parallel:
+       print("'parallel' attribute is deprecated. Use 'jobs' instead.")
```

Comment:
agree

Reply:
Actually retaining old behavior would be a bit difficult.

I have already set the default value of `jobs` to 1 which is the same with `parallel` default being False.

I think we should expect users to just stop using `parallel` entirely instead of having 2 knobs: 1 set to enable concurrency and 1 set to disable concurrency.

Reply:
If `jobs` == 1 and `parallel` is `true` then it should behave like before (i.e. set `jobs` to `0` is rule impl). If `jobs` > 0 and `parallel` is `true` then the rule should fail with an error that only one of them can be used.

Reply:
I did gave that logic a try before writing the reply. Initially I found it to be a bit confusing to new user, but let me give it a second spin next iteration.